src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,33 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int PASSWORD_LENGTH = 6;
+    private static final int MIN_AGE = 18;
+    private StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User can't be null");
+        } else if (user.getLogin() == null) {
```

Comment:
якщо в попередньому блоці є throw то далі можна не використовувати else
```suggestion
        } 
        if (user.getLogin() == null) {
```